packages/rum-core/package.json

```diff
@@ -13,7 +13,7 @@
     "karma": "karma start",
     "karma-bench": "karma start karma.bench.conf.js",
     "karma-coverage": "karma start --coverage",
-    "runUnitTests": "node ../../dev-utils/run-script.js runUnitTests true packages/rum-core",
+    "runUnitTests": "node ../../dev-utils/run-script.js runUnitTests packages/rum-core",
```

Comment:
The `true` here is for the `launchSauceConnect` parameter mentioned above!

Reply:
Yes i am aware, We don't use launchSauceConnect function anymore.. Handed it via karma-sauce-launcher.